### PR TITLE
Update XHRExampleFetch and dump the response headers

### PR DESCRIPTION
**Pull request by leeight**
Thanks for submitting a pull request! Please provide enough information so that others can review your pull request:

> **Unless you are a React Native release maintainer and cherry-picking an *existing* commit into a current release, ensure your pull request is targeting the `master` React Native branch.**

Dump the response headers so that we can check `Set-Cookie` header is working properly whether or not.

**Test plan (required)**

Manual Test.

image

**Code formatting**

Look around. Match the style of the rest of the codebase. See also the simple [style guide](https://github.com/facebook/react-native/blob/master/CONTRIBUTING.md#style-guide).

For more info, see the ["Pull Requests" section of our "Contributing" guidelines](https://github.com/facebook/react-native/blob/master/CONTRIBUTING.md#pull-requests).

### CODE REVIEW

**Comment by facebook-github-bot-9** [comment]
By analyzing the blame information on this pull request, we identified @javache and @realaboo to be potential reviewers.

**Comment by facebook-github-bot-7** [comment]
@leeight updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9687/files/592432a396b975765de16f6de5a4ef346139bc89..ad65f71925b51c93f21b4fede53da28a09e79a11)

**Comment by facebook-github-bot-1** [comment]
@leeight updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9687/files/ad65f71925b51c93f21b4fede53da28a09e79a11..18acb58ea3cdab6f3b3011fdad7fee2d62d33b50)

**Comment by leeight** [comment]
Ping @javache and @realaboo

File: Examples/UIExplorer/js/XHRExampleFetch.js [hunk 1]
```diff
@@ -63,6 +81,13 @@ class XHRExampleFetch extends React.Component {
       </View>
     ) : null;
 
+    var responseHeaders = this.responseHeaders ? (
+      <View style={{marginTop: 10}}>
+        <Text style={styles.label}>Server response Headers:</Text>
```
Comment (mkonicek) [inline Examples/UIExplorer/js/XHRExampleFetch.js]: nit: headers probably doesn't need to be capitalized?

Either:
Server response headers
or:
Server Response Headers

File: Examples/UIExplorer/js/XHRExampleFetch.js [hunk 2]
```diff
@@ -36,24 +36,42 @@ var {
 class XHRExampleFetch extends React.Component {
   state: any;
   responseURL: ?string;
+  responseHeaders: any;
```
Comment (mkonicek) [inline Examples/UIExplorer/js/XHRExampleFetch.js]: Could this be more specific? Is it an `Object`?

**Comment by mkonicek** [comment]
Thanks for adding this!

For the test plan, could you please add a screenshot showing how it renders?

**Comment by facebook-github-bot-2** [comment]
@leeight updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9687/files/18acb58ea3cdab6f3b3011fdad7fee2d62d33b50..485e49bd17602c01d181d31716c4ffdb5c5361b0)

**Comment by facebook-github-bot-8** [comment]
@leeight updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9687/files/18acb58ea3cdab6f3b3011fdad7fee2d62d33b50..485e49bd17602c01d181d31716c4ffdb5c5361b0)

**Comment by facebook-github-bot-1** [comment]
@leeight updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9687/files/485e49bd17602c01d181d31716c4ffdb5c5361b0..7404243df8ea9ba369899f4eb380eff1e6984844)

**Comment by leeight** [comment]
@mkonicek I updated this issue, and added a screenshot.

File: Examples/UIExplorer/js/XHRExampleFetch.js (hunk 1 above)
Comment (leeight) [inline Examples/UIExplorer/js/XHRExampleFetch.js]: Fixed

**Comment by facebook-github-bot-6** [comment]
@leeight updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9687/files/7404243df8ea9ba369899f4eb380eff1e6984844..3423f9a9c835af6a39f5654fad86e4734ec85e0b)

**Comment by javache** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-7** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://phabricator.intern.facebook.com/D3821145) to review internal test results.

**Pull request closed without merging**